examples/grpc-krotodc/build.gradle.kts

```
@@ -0,0 +1,27 @@
+plugins {
+    application
+}
+
+application {
+    mainClass.set("example.armeria.grpc.krotodc.MainKt")
+}
+
+dependencies {
+    implementation(project(":core"))
+    implementation(project(":grpc"))
+    compileOnly(libs.javax.annotation)
+    runtimeOnly(libs.slf4j.simple)
+
+    implementation(kotlin("stdlib-jdk8"))
+    implementation(libs.kotlin.coroutines.core)
+    implementation(libs.grpc.kotlin)
+
+    testImplementation(project(":junit5"))
+    testImplementation(libs.javax.annotation)
+    testImplementation(libs.json.unit.fluent)
+    testImplementation(libs.assertj)
+    testImplementation(libs.awaitility)
```

It seems like json unit and awaitility aren't used at all.
```suggestion
    testImplementation(libs.assertj)
```